templates/sqoop-parquet-full-load/copy-avsc.sh

```diff
@@ -1,2 +1,2 @@
-hadoop fs -mkdir {{ conf.raw_database.path }}/{{ table.destination.name }}_avro/.meta/
-hadoop fs -put -f AutoGeneratedSchema.avsc {{ conf.raw_database.path }}/{{ table.destination.name }}_avro/.meta/{{ table.destination.name }}.avsc
\ No newline at end of file
+hadoop fs -mkdir -p {{ conf.raw_database.path }}/{{ table.destination.name }}_avro/.meta/
```

Comment:
I'm not sure `-p` is the safe thing to do here. If the user has misconfigured something the application will happily keep running. What was your thought process for adding this? Can you manually create the directories first?